companies/studydrive.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "studydrive",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Studydrive GmbH",
+    "address": "Mehringdamm 32\n10961 Berlin\nDeutschland",
+    "phone": "+49 (030) 555 794 64",
+    "email": "info@studydrive.net",
+    "web": "https://www.studydrive.net",
+    "sources": [
+        "https://www.studydrive.net/datenschutz",
+        "https://www.studydrive.net/impressum"
+    ],
+    "required-elements": [
+        {
+            "desc": "E-Mail-Adresse",
+            "type": "input",
+            "optional": false
+        }
+    ],
+    "suggested-transport-medium": "email"
```

Comment:
Why is the stm email when we only have the generic `info@`.

Reply:
They explicitly ask for requests to be sent to this email.

Reply:
Sure, but our current policy is to only suggest email if the email explicitly goes to the DPO or privacy department or whatever.

Reply:
*in their privacy policy

Reply:
I know, I looked at it.

Reply:
Ok, sure.